### PR TITLE
updated openemr docker documentation

### PR DESCRIPTION
**Pull request by bradymiller**

### CODE REVIEW

**Comment by bradymiller** [comment]
@TheToolbox @jesdynf ,
Just some openemr documentation updates.

**Comment by TheToolbox** [comment]
LGTM

File: docker-images/openemr/5.0.0/README.md [hunk 1]
```diff
@@ -48,7 +49,12 @@ volumes:
 
 ## Environment Variables
 
-Setting `MYSQL_USER`, `MYSQL_ROOT_PASS`, `MYSQL_PASS`, `MYSQL_HOST`, `OE_USER`, and `OE_PASS` will do the first-time setup process without manual intervention, setting the database connection using the `MYSQL_*` variables and adding an admin user to OpenEMR using `OE_USER` and `OE_PASS`. 
+- Required environment settings for auto installation are `MYSQL_HOST` and `MYSQL_ROOT_PASS`
+    - (Note that can force `MYSQL_ROOT_PASS` to be empty by passing as 'BLANK' variable)
+- Optional settings for auto installation are:
+    - Setting db parameters `MYSQL_USER`, `MYSQL_PASS`, `MYSQL_DATABASE`
+    - Setting openemr parameters `OE_USER`, `OE_PASS`
+- Can override auto installation and force manual installation if wish by setting `MANUAL_SETUP` environment setting to 'yes'
```
Comment (jesdynf) [inline docker-images/openemr/5.0.0/README.md]: May move back to conversational English for this section in general; 'if wish' in specific is never correct.

**Comment by bradymiller** [comment]
@jesdynf @TheToolbox 
Thanks for the input. I took a stab at addressing the issues and also included instructions for the new ssl feature.
thanks, -brady

**Pull request merged**